browser/src/index.tsx

```diff
@@ -26,14 +26,16 @@ import { OverlayManager } from "./UI/Overlay/OverlayManager"
 import { ScrollBarOverlay } from "./UI/Overlay/ScrollBarOverlay"
 
 const start = (args: string[]) => {
+
     const services: any[] = []
 
     const parsedArgs = minimist(args)
     const debugPlugin = parsedArgs["debugPlugin"] // tslint:disable-line no-string-literal
 
     // Helper for debugging:
     window["UI"] = UI // tslint:disable-line no-string-literal
-
+    remote.BrowserWindow.getFocusedWindow().setFullScreen(Config.getValue<boolean>("editor.fullScreenOnStart"))
```

Comment:
Does setting this to `false` have any behavioral change than just not calling it at at all? Just curious!